notebooks/00_quick_start/xdeepfm_movielens.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<p>is it possible to use the movielens data loader and pandas_df_utils.libffm_converter to generate necessary files?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
will try. To convert 'date', libffm_converter may not generalized enough.